packages/twenty-front/src/modules/ui/input/components/Select.tsx

```diff
@@ -160,6 +161,12 @@ export const Select = <Value extends SelectValue>({
           }
           dropdownComponents={
             <DropdownContent widthInPixels={dropDownMenuWidth}>
+              {isDropdownOpen && selectedOption && !searchInputValue && (
```

Comment:
We shouldn't need to use isDropdownOpen because dropdownComponents is not mounted when isDropdownOpen is false